src/foram3d/chamber.ts

```diff
@@ -10,6 +10,9 @@ module Foram3D {
     private static WIDTH_SEGMENTS:  number = 32;
     private static HEIGHT_SEGMENTS: number = 32;
 
+    private static APERTURE_MARKER_COLOR:       number = 0x000000;
```

Comment:
Jednak tego nie poprawię, bo musimy jeszcze pomnożyć `SIZE_FACTOR` przez `this.radius` i wychodzi gorzej nie jest aktualnie :-1: